docs/features/tcp-health-probe.md

```diff
@@ -35,10 +35,30 @@ public void ConfigureServices(IServiceCollection services)
 
 ## Configuration
 
-To make the TCP health check fully functional, you'll  require some configuration values. 
+The TCP probe allows several additional configuration options.
 
-| Configuration key   | Usage        | Type  | Description                                                         |
-| ------------------- | ------------ | ----- | ------------------------------------------------------------------- |
-| `ARCUS_HEALTH_PORT` | **required** | `int` | The TCP port on which the health report is exposed.                 |
+```csharp
+public void ConfigureServices(IServiceCollection services)
+{
+    // Add TCP health probe with or whitout extra health checks.
+    services.AddTcpHealthProbes(
+        configureTcpListenerOptions: options =>
+        {
+            // Configure the configuration key on which the health report is exposed.
+            options.TcpPortConfigurationKey = "MyConfigurationKey";
+
+            // Configure how the health report should be serialized.
+            options.HealthReportSerializer = new MyHealthReportSerializer();
+        });
+}
+
+public class MyHealthReportSerializer : IHealthReportSerializer
```

Comment:
I would move this to a second paragraph so that we have:
1. A section on minimal required
2. A section on bringing your own serialization

Reply:
? But isn't that already the case? WIth the `Usage` section?

Reply:
I've missed that one, sorry!